dashboard/test/ui/features/stage_lock.feature

```diff
@@ -36,7 +36,8 @@ Scenario: Lock settings for students
   When I sign out
   And I sign in as "Teacher_bobby"
   And I am on "http://studio.code.org/s/allthethings"
-  And I select the first section
+  # Wait until detail view loads
+  And I wait until element "span:contains(Lesson 1: Jigsaw)" is visible
```

Comment:
The unit overview page starts in summary view, but then switches over to detail view later in the page load sequence.